server/chat-plugins/help.ts

```diff
@@ -335,7 +345,7 @@ export const commands: ChatCommands = {
 		toggle(target, room, user) {
 			if (!room) return this.requiresRoom();
 			const helpRoom = Answerer.getRoom();
-			if (!helpRoom) return this.errorReply(`There is no room configured for use of this filter.`);
+			if (!helpRoom) HelpResponder.roomNotFound();
```

Comment:
Correct me if I'm wrong, but since `roomNotFound` is `throw`ing, this results in a crash message in dev and an email, which really seems like a bad idea for something simply triggerable via a command. If thats the case a plain error message to the command's user would be better.

Reply:
it throws a Chat.ErrorMessage, which is transformed into an `errorReply` by the command parser.

Reply:
I leveraged a throw since the `roomNotFound` can be used in other functions that are called within commands, in that case, and it's easier to just use one thing.